apps/fluent-tester/src/FluentTester/TestComponents/Menu/MenuTest.tsx

```diff
@@ -55,8 +56,9 @@ const MenuCheckmarks: React.FunctionComponent = () => {
         </MenuTrigger>
         <MenuPopover>
           <MenuList>
-            <MenuItem content="A MenuItem" />
-            <MenuItemCheckbox content="A MenuItem" />
+            <MenuItem content="A plain MenuItem" />
```

Comment:
This menu purposefully doesn't set `hasCheckmarks` so that you can see the visual difference?

Reply:
Yeah, the collapsing hides the trigger text but it says "Some checkmark items without alignment"

Reply:
Ah derp, didn't catch that. Thanks!